CodeStarterCamp_Week2/main.swift

```diff
@@ -1,33 +1,35 @@
 //
-//  main.swift
+//  main5.swift
 //  CodeStarProject
 //
-//  Created by 유지완 on 2023/08/27.
+//  Created by 유지완 on 2023/08/31.
+
+
 
-//STEP2
 import Foundation
 
-let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
-var lottoNumders: Set<Int> = []
+let myLottoNumber: [Int] = [1, 2, 3, 4, 5, 6]
+var lottoNumber: Set<Int> = []
+
+createLottoNumder()
+
+var lottoResults: [Int: Set<Int>] = [:] // 각 회차의 로또 당첨 번호를 저장할 Dictionary
```

Comment:
Set 타입을 Array 타입으로 덮어씌울 수 있으신걸 아시나요?
또한 요구사항에 보시면 "O회차" 라는 String 값을 키값으로 사용하고 있습니다.
항상 요구사항 명세를 꼼꼼히 확인할 수 있도록 해주세요

또한 불필요한 주석은 꼭 지워주세요 이전 스텝부터 계속 주석이 남아있네요
```suggestion
var lottoResults: [String: [Int]] = [:] 
```